src/render.ts

```diff
@@ -146,17 +150,31 @@ export async function renderPlan({
   planfile: string
   terraformCommand: string
   workingDirectory: string
-}): Promise<RenderedPlan> {
+}): Promise<Array<RenderedPlan>> {
   const options = {
     cwd: workingDirectory,
     silent: true
   }
-  const structuredPlanfile = await exec
-    .getExecOutput(terraformCommand, ['show', '-json', planfile], options)
-    .then((output) => JSON.parse(output.stdout))
-    .then((json) => parsePlanfileJSON(json))
   const humanReadablePlanfile = await exec
     .getExecOutput(terraformCommand, ['show', '-no-color', planfile], options)
     .then((output) => output.stdout)
-  return internalRenderPlan(structuredPlanfile, humanReadablePlanfile)
+  try {
```

Comment:
Can we add a code comment above this try-except block why we do this "dance"? :smile:

Reply:
I broke try and catch blocks into its own functions. I hope it makes it more clear. Also added a comment